src/js/hooks/useArrowsNavigation.jsx

```diff
@@ -0,0 +1,108 @@
+import navigationKey from 'consts/navigationKey';
+
+const useArrowsNavigation = ({
+  newRowFocusColumns,
+  existingRowFocusColumns,
+  tableData,
+  setFocusId,
+  setFocusIndex,
+  addNewRow,
+}) => {
+  const isNewRow = (row) => !row?.id || row?.id?.includes('newRow') || row?.id === null;
```

Comment:
It should be a function passed as an argument. We are not sure whether the new row will be determined in the same way in every case.

+ Please add a comment explaining what every argument is and what their responsibility is. It will be helpful for someone who will be using this hook in the future, but at this moment, I don't know what I should pass for at least half of those arguments.